callgraph/render.py

```diff
@@ -53,6 +53,10 @@ def PrintDot(call_graph, out_file=sys.stdout, log_file=sys.stderr, show_all_call
 
         attributes.append("label=<{}>".format("<br/>".join(label_lines)))
 
+        # Minimum width and height of each node proportional to the number of lines contained (self.loc)
+        attributes.append("width={}".format(node.node_width))

```

Comment:
I suggest to round up to the nearest integer, to avoid having a lot of useless digits in the resulting dot file.

Reply:
Rounded up to one decimal digit so as to preserve ellipses regularity.